src/providers/proxy/proxy_ipnetworks.c

```diff
@@ -334,7 +334,7 @@ get_net_byaddr(struct proxy_resolver_ctx *ctx,
     if (ret == ENOENT) {
         /* Not found, make sure we remove it from the cache */
         DEBUG(SSSDBG_TRACE_INTERNAL, "Network [%s] not found, removing from "
-              "cache\n", name);
+              "cache\n", search_addrstr);
```

Comment:
The same comment about `DEBUG(SSSDBG_TRACE_INTERNAL, "Network [%s] found, saving into "` below.

Reply:
Same response: :smiley: